ui/webui/src/components/HeaderKebab.jsx

```diff
@@ -135,40 +130,43 @@ export const HeaderKebab = ({ reportLinkURL, isConnected }) => {
         setIsReportIssueOpen(true);
     };
 
-    const dropdownItems = [
```

Comment:
This change is unnecessary, children can be n array of react nodes. It's probably just not documented well.